src/index.html

```diff
@@ -1,21 +1,57 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta 
-      name="viewport"
-      content="
-        width=device-width, 
-        user-scalable=no, 
-        initial-scale=1.0, 
-        maximum-scale=1.0, 
-        minimum-scale=1.0
-      "
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/index.css">
   </head>
-  <body>
-    <h1>Product cards</h1>
+
+  <body class="page">
+    <article class="page__card card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="iMac"
+        class="card__image"
+      >
+
+      <h1 class="card__product-name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+
+      <p class="card__product-code">Product code: 195434</p>
+
+      <div class="card__review">
+        <div class="card__stars stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <span class="card__count">Reviews: 5</span>
+      </div>
+
+      <div class="card__price">
+        Price:
+        <p class="card__price-tag">$2,199</p>
+      </div>
+
+      <a
+        href="/"
+        class="card__button"
+        data-qa="hover">
+        Buy
+      </a>
+    </article>
```

Comment:
```suggestion
    <div class="page__card card" data-qa="card">
      <img
        src="images/imac.jpeg"
        alt="iMac"
        class="card__image"
      >

      <h1 class="card__product-name">
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </h1>

      <p class="card__product-code">Product code: 195434</p>

      <div class="card__review">
        <div class="card__stars stars stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>
        <span class="card__count">Reviews: 5</span>
      </div>

      <div class="card__price">
        Price:
        <p class="card__price-tag">$2,199</p>
      </div>

      <a
        href="/"
        class="card__button"
        data-qa="hover">
        Buy
      </a>
    </div>
```

As output is not wrong, but <div> is more suitable as a container for general content than <article>.
We should choose the tag according to the content element and it's semantic.

Articles are more suitable for forums, news papers, blog, etc.

Something to read: https://bitsofco.de/sectioning-content-in-html5/#:~:text=So%20which%20should%20you%20use,then%20use%20an%20.